.github/workflows/release-package.yml

```diff
@@ -0,0 +1,39 @@
+name: Release NPM package
+
+on:
+    pull_request:
+        branches:
+            - develop
+        types: [closed]
+
+jobs:
+    auto-release:
+        if: github.event.pull_request.merged == true
+        name: Automated package release
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Clone the repository
+              uses: actions/checkout@v3
```

Comment:
@rahulfoc1990 is updating the actions/checkout to use the @v4 instead, might want to go with that https://oat-sa.atlassian.net/browse/OPS-534